src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/QualificationsResourceMapper.kt

```diff
@@ -55,6 +61,21 @@ abstract class QualificationsResourceMapper {
   fun toQualifications(achievedQualifications: List<AchievedQualification>): List<Qualification> =
     achievedQualifications.map { toQualification(it) }
 
+  fun toAchievedQualificationResponse(qualification: Qualification): AchievedQualificationResponse =
```

Comment:
New mapper method to map from `Qualification` (the domain object) to `AchievedQualificationResponse` (the REST API response model object)